thorlcr/graph/thgraphmaster.cpp

```diff
@@ -2857,8 +2858,28 @@ void CMasterGraph::getStats(IStatisticGatherer &stats)
 
             StatsActivityScope scope(stats, id);
             activity->getActivityStats(stats);
+            totalDiskAccessCost += activity->getDiskAccessCost();
         }
     }
+    stats.addStatistic(StCostFileAccess, totalDiskAccessCost);
+}
+
+cost_type CMasterGraph::getDiskAccessCost()
+{
```

Comment:
I agree it doesn't sound right to be calculating the costs more than once.  Are you recording the same stat twice?  You definitely shouldn't need to do that.
There may be an issue that graph and subgraph stats should be recorded within the workunit directly, rather than the subgraph blob, but if so that is a general issue and should be handled a different way.  (Stats from the workunit and subgraph blobs are merged when reporting in wudetails, so shouldn't need to be duplicated).

Reply:
Duplication now removed.